Client/src/Utils/Theme/globalTheme.js

```diff
@@ -237,6 +237,11 @@ const baseTheme = (palette) => ({
 							WebkitTextFillColor: "unset",
 						},
 					},
+					"& .MuiInputBase-input:-webkit-autofill": {
+						transition: "background-color 5000s ease-in-out 0s", 
+						WebkitBoxShadow: `0 0 0px 1000px ${theme.palette.background.main} inset`, 
```

Comment:
Can you explain the use of BoxShadow here?  Is it being used to set background color?  This seems to be a very roundabout approach to resolving the issue 🤔

Reply:
Hey ! The box-shadow here (specifically WebkitBoxShadow) to set a background color directly, but rather to override the default background color applied by browsers to **auto filled** input fields.

### Reason being:
Browsers, particularly Chrome and some **WebKit**-based ones, apply a default background color to **auto filled** fields. This color is not easily overridden by standard CSS properties like background-color.
Using the **-webkit-box-shadow** property with an inset shadow of a large size allows developers to **"paint over"** the browser's default styling. By specifying the desired background color **(${theme.palette.background.main})**, the **autofilled** field effectively displays the intended color.

This workaround is especially useful when browsers don’t allow direct manipulation of the autofill background color using the standard background-color property.

Reply:
It works perfectly with this, as demonstrated here:
![image](https://github.com/user-attachments/assets/d7e3bc1c-1629-4973-88d4-c706a5733b41)
![image](https://github.com/user-attachments/assets/ea246f79-fee2-42f1-a275-df02c30f2a7f)

Additionally, the pale white background in the text input field was caused by Chrome's autofill feature. To override this, we can't simply use background-color, as it's ineffective once the autofill has applied its own background. Instead, we can use a roundabout approach with **-webkit-box-shadow** to cover the autofill background and apply the desired styling.

Reply:
Also, regarding the transition property; it is intended to animate changes to the background-color of the input. The duration of 5000s is so long that the transition is essentially "frozen," meaning it won't be noticeable. This long duration is used to prevent the browser from abruptly changing the background color when the input is auto filled. Instead, the background color change becomes so gradual (taking thousands of seconds) that it effectively doesn't change visibly at all.

The transition ensures that if the background color of the input were to change (such as during autofill), the change would be gradual. However, because the background change is **"frozen"**, it remains imperceptible. Meanwhile, the **WebkitBoxShadow** is used to hide any background color that might appear as a result of autofill, ensuring the auto filled input looks consistent with the rest of the form.

Together, both properties are used to address the nuisance of the default autofill background color, ensuring a smooth, consistent appearance for the autofilled input fields without disrupting the design.

Reply:
Feel free to let me know if you need any further details regarding the PR. You can also refer to the following links, which demonstrate a similar approach:
[NextUI GitHub Issue #1346](https://github.com/nextui-org/nextui/issues/1346)
[A Trick to Change the Autocomplete Background Color](https://dev.to/atndesign/a-trick-to-change-the-autocomplete-background-color-21ll)

Reply:
If you have the time for this sir, please review it once again !

Reply:
Hi @saksham-malhotra-27 , thanks for your explanation here.  I'm not a front end expert, so I'll leave the detailed review to @marcelluscaio and @jennifer-gan.

Reply:
Hey ! No problem at all—happy to provide context.